### PR TITLE
Added winston logger

### PR DESCRIPTION
**Pull request by igna92ts**
Added logger and some basic logging channels

### CODE REVIEW

**Comment by gonziesc** [comment]
@mpizzagalli @mishuagopian @igna92ts

File: app.js [hunk 1]
```diff
@@ -62,8 +63,9 @@ const init = () => {
       app.use(rollbar.errorHandler());
 
       app.listen(port);
-      console.log(`Listening on port: ${port}`); // eslint-disable-line
+
+      logger.info(`Listening on port: ${port}`); // eslint-disable-line
```
Comment (mpizzagalli) [inline app.js]: I think we can remove the `eslint-disable-line`

File: app.js [hunk 2]
```diff
@@ -62,8 +63,9 @@ const init = () => {
...
     })
-    .catch(console.log); // eslint-disable-line
+    .catch(logger.error); // eslint-disable-line
```
Comment (mpizzagalli) [inline app.js]: same here

File: package.json [hunk 3]
```diff
@@ -43,12 +43,14 @@
     "bcrypt": "^1.0.1",
     "body-parser": "^1.14.2",
     "express": "^4.9.4",
-    "jwt-simple": "^0.5.1",
+    "jwt-simple": "^0.3.1",
```
Comment (mpizzagalli) [inline package.json]: Why are we downgrading this version?

**Comment by gonziesc** [comment]
@mishuagopian

**Pull request merged**